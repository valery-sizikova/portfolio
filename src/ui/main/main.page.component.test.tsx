import * as React from "react";
import { shallow } from 'enzyme';

import MainPageComponent from "./main.page.component";

describe('Main page component', () => {
  it('renders snapshots', () => {
    const wrapper = shallow(<MainPageComponent />);
    expect(wrapper).toMatchSnapshot()
  })
})