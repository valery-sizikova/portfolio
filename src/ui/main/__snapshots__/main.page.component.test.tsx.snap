// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page component renders snapshots 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MainPageComponent />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Menu
          className="navbar"
          isVertical={true}
        >
          <MenuItem>
            <Link
              activeClass="active"
              duration={1000}
              offset={-100}
              onSetActive={[Function]}
              smooth={true}
              spy={true}
              to="section_1"
            >
              Section 1
            </Link>
          </MenuItem>
          <MenuItem>
            <Link
              activeClass="active"
              duration={1000}
              offset={-100}
              onSetActive={[Function]}
              smooth={true}
              spy={true}
              to="section_2"
            >
              Section 2
            </Link>
          </MenuItem>
          <MenuItem>
            <Link
              activeClass="active"
              duration={1000}
              offset={-100}
              onSetActive={[Function]}
              smooth={true}
              spy={true}
              to="section_3"
            >
              Section 3
            </Link>
          </MenuItem>
          <MenuItem>
            <Link
              activeClass="active"
              duration={1000}
              offset={-100}
              onSetActive={[Function]}
              smooth={true}
              spy={true}
              to="section_4"
            >
              Section 4
            </Link>
          </MenuItem>
        </Menu>,
        Array [
          <Element
            className="element"
            name="section_1"
          >
            <section
              className="section-big"
            >
              <h4>
                Section 1
              </h4>
            </section>
          </Element>,
          <Element
            className="element"
            name="section_2"
          >
            <section
              className="section-big"
            >
              <h4>
                Section 2
              </h4>
            </section>
          </Element>,
          <Element
            className="element"
            name="section_3"
          >
            <section
              className="section-big"
            >
              <h4>
                Section 3
              </h4>
            </section>
          </Element>,
          <Element
            className="element"
            name="section_4"
          >
            <section
              className="section-big"
            >
              <h4>
                Section 4
              </h4>
            </section>
          </Element>,
        ],
      ],
      "className": "main-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <MenuItem>
              <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_1"
              >
                Section 1
              </Link>
            </MenuItem>,
            <MenuItem>
              <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_2"
              >
                Section 2
              </Link>
            </MenuItem>,
            <MenuItem>
              <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_3"
              >
                Section 3
              </Link>
            </MenuItem>,
            <MenuItem>
              <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_4"
              >
                Section 4
              </Link>
            </MenuItem>,
          ],
          "className": "navbar",
          "isVertical": true,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "link_to_section_1",
            "nodeType": "function",
            "props": Object {
              "children": <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_1"
              >
                Section 1
              </Link>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "activeClass": "active",
                "children": "Section 1",
                "duration": 1000,
                "offset": -100,
                "onSetActive": [Function],
                "smooth": true,
                "spy": true,
                "to": "section_1",
              },
              "ref": null,
              "rendered": "Section 1",
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "link_to_section_2",
            "nodeType": "function",
            "props": Object {
              "children": <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_2"
              >
                Section 2
              </Link>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "activeClass": "active",
                "children": "Section 2",
                "duration": 1000,
                "offset": -100,
                "onSetActive": [Function],
                "smooth": true,
                "spy": true,
                "to": "section_2",
              },
              "ref": null,
              "rendered": "Section 2",
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "link_to_section_3",
            "nodeType": "function",
            "props": Object {
              "children": <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_3"
              >
                Section 3
              </Link>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "activeClass": "active",
                "children": "Section 3",
                "duration": 1000,
                "offset": -100,
                "onSetActive": [Function],
                "smooth": true,
                "spy": true,
                "to": "section_3",
              },
              "ref": null,
              "rendered": "Section 3",
              "type": [Function],
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "link_to_section_4",
            "nodeType": "function",
            "props": Object {
              "children": <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_4"
              >
                Section 4
              </Link>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "activeClass": "active",
                "children": "Section 4",
                "duration": 1000,
                "offset": -100,
                "onSetActive": [Function],
                "smooth": true,
                "spy": true,
                "to": "section_4",
              },
              "ref": null,
              "rendered": "Section 4",
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "section_section_1",
        "nodeType": "class",
        "props": Object {
          "children": <section
            className="section-big"
          >
            <h4>
              Section 1
            </h4>
          </section>,
          "className": "element",
          "name": "section_1",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h4>
              Section 1
            </h4>,
            "className": "section-big",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Section 1",
            },
            "ref": null,
            "rendered": "Section 1",
            "type": "h4",
          },
          "type": "section",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "section_section_2",
        "nodeType": "class",
        "props": Object {
          "children": <section
            className="section-big"
          >
            <h4>
              Section 2
            </h4>
          </section>,
          "className": "element",
          "name": "section_2",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h4>
              Section 2
            </h4>,
            "className": "section-big",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Section 2",
            },
            "ref": null,
            "rendered": "Section 2",
            "type": "h4",
          },
          "type": "section",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "section_section_3",
        "nodeType": "class",
        "props": Object {
          "children": <section
            className="section-big"
          >
            <h4>
              Section 3
            </h4>
          </section>,
          "className": "element",
          "name": "section_3",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h4>
              Section 3
            </h4>,
            "className": "section-big",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Section 3",
            },
            "ref": null,
            "rendered": "Section 3",
            "type": "h4",
          },
          "type": "section",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": "section_section_4",
        "nodeType": "class",
        "props": Object {
          "children": <section
            className="section-big"
          >
            <h4>
              Section 4
            </h4>
          </section>,
          "className": "element",
          "name": "section_4",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h4>
              Section 4
            </h4>,
            "className": "section-big",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Section 4",
            },
            "ref": null,
            "rendered": "Section 4",
            "type": "h4",
          },
          "type": "section",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Menu
            className="navbar"
            isVertical={true}
          >
            <MenuItem>
              <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_1"
              >
                Section 1
              </Link>
            </MenuItem>
            <MenuItem>
              <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_2"
              >
                Section 2
              </Link>
            </MenuItem>
            <MenuItem>
              <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_3"
              >
                Section 3
              </Link>
            </MenuItem>
            <MenuItem>
              <Link
                activeClass="active"
                duration={1000}
                offset={-100}
                onSetActive={[Function]}
                smooth={true}
                spy={true}
                to="section_4"
              >
                Section 4
              </Link>
            </MenuItem>
          </Menu>,
          Array [
            <Element
              className="element"
              name="section_1"
            >
              <section
                className="section-big"
              >
                <h4>
                  Section 1
                </h4>
              </section>
            </Element>,
            <Element
              className="element"
              name="section_2"
            >
              <section
                className="section-big"
              >
                <h4>
                  Section 2
                </h4>
              </section>
            </Element>,
            <Element
              className="element"
              name="section_3"
            >
              <section
                className="section-big"
              >
                <h4>
                  Section 3
                </h4>
              </section>
            </Element>,
            <Element
              className="element"
              name="section_4"
            >
              <section
                className="section-big"
              >
                <h4>
                  Section 4
                </h4>
              </section>
            </Element>,
          ],
        ],
        "className": "main-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <MenuItem>
                <Link
                  activeClass="active"
                  duration={1000}
                  offset={-100}
                  onSetActive={[Function]}
                  smooth={true}
                  spy={true}
                  to="section_1"
                >
                  Section 1
                </Link>
              </MenuItem>,
              <MenuItem>
                <Link
                  activeClass="active"
                  duration={1000}
                  offset={-100}
                  onSetActive={[Function]}
                  smooth={true}
                  spy={true}
                  to="section_2"
                >
                  Section 2
                </Link>
              </MenuItem>,
              <MenuItem>
                <Link
                  activeClass="active"
                  duration={1000}
                  offset={-100}
                  onSetActive={[Function]}
                  smooth={true}
                  spy={true}
                  to="section_3"
                >
                  Section 3
                </Link>
              </MenuItem>,
              <MenuItem>
                <Link
                  activeClass="active"
                  duration={1000}
                  offset={-100}
                  onSetActive={[Function]}
                  smooth={true}
                  spy={true}
                  to="section_4"
                >
                  Section 4
                </Link>
              </MenuItem>,
            ],
            "className": "navbar",
            "isVertical": true,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "link_to_section_1",
              "nodeType": "function",
              "props": Object {
                "children": <Link
                  activeClass="active"
                  duration={1000}
                  offset={-100}
                  onSetActive={[Function]}
                  smooth={true}
                  spy={true}
                  to="section_1"
                >
                  Section 1
                </Link>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "activeClass": "active",
                  "children": "Section 1",
                  "duration": 1000,
                  "offset": -100,
                  "onSetActive": [Function],
                  "smooth": true,
                  "spy": true,
                  "to": "section_1",
                },
                "ref": null,
                "rendered": "Section 1",
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "link_to_section_2",
              "nodeType": "function",
              "props": Object {
                "children": <Link
                  activeClass="active"
                  duration={1000}
                  offset={-100}
                  onSetActive={[Function]}
                  smooth={true}
                  spy={true}
                  to="section_2"
                >
                  Section 2
                </Link>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "activeClass": "active",
                  "children": "Section 2",
                  "duration": 1000,
                  "offset": -100,
                  "onSetActive": [Function],
                  "smooth": true,
                  "spy": true,
                  "to": "section_2",
                },
                "ref": null,
                "rendered": "Section 2",
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "link_to_section_3",
              "nodeType": "function",
              "props": Object {
                "children": <Link
                  activeClass="active"
                  duration={1000}
                  offset={-100}
                  onSetActive={[Function]}
                  smooth={true}
                  spy={true}
                  to="section_3"
                >
                  Section 3
                </Link>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "activeClass": "active",
                  "children": "Section 3",
                  "duration": 1000,
                  "offset": -100,
                  "onSetActive": [Function],
                  "smooth": true,
                  "spy": true,
                  "to": "section_3",
                },
                "ref": null,
                "rendered": "Section 3",
                "type": [Function],
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "link_to_section_4",
              "nodeType": "function",
              "props": Object {
                "children": <Link
                  activeClass="active"
                  duration={1000}
                  offset={-100}
                  onSetActive={[Function]}
                  smooth={true}
                  spy={true}
                  to="section_4"
                >
                  Section 4
                </Link>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "activeClass": "active",
                  "children": "Section 4",
                  "duration": 1000,
                  "offset": -100,
                  "onSetActive": [Function],
                  "smooth": true,
                  "spy": true,
                  "to": "section_4",
                },
                "ref": null,
                "rendered": "Section 4",
                "type": [Function],
              },
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "section_section_1",
          "nodeType": "class",
          "props": Object {
            "children": <section
              className="section-big"
            >
              <h4>
                Section 1
              </h4>
            </section>,
            "className": "element",
            "name": "section_1",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h4>
                Section 1
              </h4>,
              "className": "section-big",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Section 1",
              },
              "ref": null,
              "rendered": "Section 1",
              "type": "h4",
            },
            "type": "section",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "section_section_2",
          "nodeType": "class",
          "props": Object {
            "children": <section
              className="section-big"
            >
              <h4>
                Section 2
              </h4>
            </section>,
            "className": "element",
            "name": "section_2",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h4>
                Section 2
              </h4>,
              "className": "section-big",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Section 2",
              },
              "ref": null,
              "rendered": "Section 2",
              "type": "h4",
            },
            "type": "section",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "section_section_3",
          "nodeType": "class",
          "props": Object {
            "children": <section
              className="section-big"
            >
              <h4>
                Section 3
              </h4>
            </section>,
            "className": "element",
            "name": "section_3",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h4>
                Section 3
              </h4>,
              "className": "section-big",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Section 3",
              },
              "ref": null,
              "rendered": "Section 3",
              "type": "h4",
            },
            "type": "section",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "section_section_4",
          "nodeType": "class",
          "props": Object {
            "children": <section
              className="section-big"
            >
              <h4>
                Section 4
              </h4>
            </section>,
            "className": "element",
            "name": "section_4",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h4>
                Section 4
              </h4>,
              "className": "section-big",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Section 4",
              },
              "ref": null,
              "rendered": "Section 4",
              "type": "h4",
            },
            "type": "section",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
